tests/unit-tests/test_embed.py

```diff
@@ -191,6 +191,25 @@ def test_input_layer4(dev):
         # get the test dummy distributed graph
         g, _ = generate_dummy_dist_graph(tmpdirname)
 
+    # Test 0: Inputs are [{}, {}, ...], and output should be [{}, {}, ...] too.
+    edge_feat_size = {('n0', 'r0', 'n1'): 0, ('n0', 'r1', 'n1'): 0, ('n1', 'r2', 'n2'): 0}
+    edge_input_layer = GSEdgeEncoderInputLayer(g, edge_feat_size, 2, activation=None)
+    assert len(edge_input_layer.input_projs) == 0
+    edge_input_layer = edge_input_layer.to(dev)
+
+    block_edge_feat_list = [{}, {}]
+    embed = edge_input_layer(block_edge_feat_list)
+    assert len(embed) == len(block_edge_feat_list)
+    assert any(embed) == False
+
+    edge_feat_size = get_edge_feat_size(g, {('n0', 'r0', 'n1'): ['feat'], \
+                                            ('n0', 'r1', 'n1'): ['feat']})
```

Comment:
The edge_feat_size here is [{}, {}]?

Reply:
No. It will be `('n0', 'r0', 'n1'): 2, ('n0', 'r1', 'n1'): 2`. This test setting is when `edge_input_layer` has weights, but the input is an empty dict list, the input layer still works.